tower-scripts/bin/aos-cd-cluster-smoke-test.sh

```diff
@@ -29,7 +29,7 @@ if [ "$?" != "0" ]; then
 fi
 
 if oc get nodes -l type=infra | grep -e SchedulingDisabled -e NotReady; then
```

Comment:
This is as fragile as `oc get pods | grep Running`

Reply:
Why don't we make the node schedulable if it's unschedulable? Also, I would retry for some time on NotReady.

Reply:
I think that fixing stuff is not what the smoke test is for - if there's a problem, we want to highlight it, not try to fix it here as part of the test.

So, the question is if we should fail or not based on having *any* non-schedulable or NotReady node. Probably we don't... so we could just write down the warning; but then no one would read it probably (buried down in logs).

I'm thinking we should just remove this part of the smoke test (check for NotReady/unschedulable nodes). Note that the cluster status post-upgrade is already reported by the upgrade job, and this includes output of "oc get node".